_posts/2017-03-23-omero-5.2.8.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: [SECURITY] Release of OMERO 5.2.8
```

Comment:
This page doesn't seem to be showing up, I wonder if the [SECURITY] in the title here is why - can you remove the square brackets please?